components/transactions/TransactionItem.js

```diff
@@ -108,7 +108,7 @@ const getExpenseStatusTag = (expense, isRefund, isRefunded) => {
   if (isRefunded) {
     expenseStatusLabel = expenseStatus.REFUNDED;
```

Comment:
Isn't this now `undefined`?

Reply:
Can you double-check and test?

![image](https://user-images.githubusercontent.com/1556356/172597395-302cedc2-2ada-4c9f-8a9b-b358d528fcd6.png)

![image](https://user-images.githubusercontent.com/1556356/172597547-324c4540-eb80-4865-b2f8-de65d725a21a.png)

Reply:
@Betree : Ops, I am so sorry, I somehow got confused here, yes you are right, there's an error on using a non-existent constant here. I've corrected it. 👍🏽